src/PHPixie/Debug.php

```diff
@@ -39,6 +74,17 @@ public function dumpLog($withTitle = true, $withTraceArguments = true, $shortVal
         return $log;
     }
     
+    /**
+     * Displaying code where this method was 
```

Comment:
This method does not actually display anything, it returns a formatted string representing an exception.